packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [55.1.0]
```

Comment:
Minor, did you mean `userLevelFee` instead of `userLevelFee` below?

Reply:
Release seems to have mostly fixes do we want to use a patch instead ?